vtex-apps/return-app
# Handle errors

What this PR changes:
Improve error handling and use splunk logger method in order to see on the platform.


How to test it:
[this workspace](https://zekiqa--vtexspain.myvtex.com/account#/my-returns).
Add a `throw new Error('any message')` on any defined route go to to the given workspace reload the page and then you can see the error on splunk.
`receiveDocuments.ts` is a good file to test because it loads on the root of the app.
